src/foam/nanos/fs/fileDropZone/FilePreview.js

```diff
@@ -0,0 +1,37 @@
+foam.CLASS({
+  package: 'foam.nanos.fs.fileDropZone',
+  name: 'FilePreview',
+  extends: 'foam.u2.View',
+
+  documentation: 'iframe for file preview',
+
+  properties: [
```

Comment:
How is the sizing controlled?

Reply:
Added css. Size limited by MultiView container